### PR TITLE
add validation output

### PR DESCRIPTION
**Pull request by pschoffer**
## Description

We already have some validations in place before each command. This change makes the validation visible on every run. This should help troubleshoot problems by users and support.

## Steps to Test

```
$ ./dist/bin/vip-dev-env-info.js 
Running validation steps...
✓ Check for docker instalation 
✓ Check for docker-compose instalation 
✓ Check DNS resolution 
 SLUG              vip-local                                                                      
 LOCATION          /home/pavel/.local/share/vip/dev-environment/vip-local                         
...
```

### CODE REVIEW

File: src/lib/dev-environment/dev-environment-cli.js [hunk 1]
```diff
@@ -95,14 +95,31 @@ const verifyDNSResolution = ( slug: string ) => {
 		} );
 	} );
 };
-
+const VALIDATION_STEPS = [
+	{ id: 'docker', name: 'Check for docker instalation' },
```
Comment (sjinks) [inline src/lib/dev-environment/dev-environment-cli.js]:
```suggestion
	{ id: 'docker', name: 'Check for docker installation },
```

File: src/lib/dev-environment/dev-environment-cli.js [hunk 2]
```diff
@@ -95,14 +95,31 @@ const verifyDNSResolution = ( slug: string ) => {
...
+	{ id: 'compose', name: 'Check for docker-compose instalation' },
```
Comment (sjinks) [inline src/lib/dev-environment/dev-environment-cli.js]:
```suggestion
	{ id: 'compose', name: 'Check for docker-compose installation },
```

File: src/lib/dev-environment/dev-environment-cli.js [hunk 3]
```diff
@@ -95,14 +95,31 @@ const verifyDNSResolution = ( slug: string ) => {
...
 	} catch ( exception ) {
 		throw new UserError( exception.message );
 	}
+	progressTracker.stepSuccess( 'docker' );
+	progressTracker.stepSuccess( 'compose' );
+	progressTracker.print();
+
 	await verifyDNSResolution( slug );
+
+	progressTracker.stepSuccess( 'dns' );
```
Comment (sjinks) [inline src/lib/dev-environment/dev-environment-cli.js:119]: I didn't look into `progressTracker` but I would like to confirm that we don't need

```js
progressTracker.stepRunning( 'dns' );
```

before `await verifyDNSResolution( slug );`

**Review by sjinks — CHANGES_REQUESTED** [review]

File: src/lib/dev-environment/dev-environment-cli.js (hunk 3 above)
Comment (pschoffer) [reply src/lib/dev-environment/dev-environment-cli.js:119]: Thanks for the fixes!

Yeah `progressTracker.stepSuccess( <stepId> );` moves it to the next step so no need to call stepRunning.

**Review by sjinks — APPROVED** [review]

**Pull request merged**